static/temp/foods.json

```diff
@@ -0,0 +1,250 @@
+[
+  {
+    "name": "Banana ripe, slightly ripe, raw",
+    "categoryName": "Fruit and Fruit Juices",
+    "categoryIcon": "EggAltOutlinedIcon",
+    "overview": {
+      "info": "This is a cool food",
+      "NutrientScore": ["A", "B", "C", "D"],
```

Comment:
1. Keys should be camelCase. `nutrientScore`.
2. A food will have only one of the possible values - `A`, `B`, `C`, `D`, or `E`. Hence, the final object should be

```text
nutrientScore: "B"
```

Reply:
done